niworkflows/interfaces/bids.py

```diff
@@ -19,10 +23,15 @@
 )
 from nipype.interfaces.io import add_traits
 from templateflow.api import templates as _get_template_list
-from ..utils.bids import BIDS_NAME, _init_layout
+from ..utils.bids import _init_layout
 from ..utils.images import overwrite_header
 from ..utils.misc import splitext as _splitext, _copy_any
 
+_pybids_spec = loads(
+    Path(_pkgres("niworkflows", "data/derivatives.json")).read_text()
+)
+BIDS_DERIV_ENTITIES = tuple({e["name"] for e in _pybids_spec["entities"]})
```

Comment:
does this need to be a tuple? it seems this is always being converted to a set

Reply:
I wanted it to be immutable, although I could've used `frozenset`. Let me know if that is relevant to change.